tests/test_all_fields.py

```diff
@@ -0,0 +1,19 @@
+"""
+Test that with no fields selected for a stream automatic fields are still replicated
+"""
+
+from tap_tester import runner, menagerie, connections
+from tap_tester.base_suite_tests.all_fields_test import AllFieldsTest
+from tap_tester.logger import LOGGER
```

Comment:
Wondering if we need all these imports?  For tests where we aren't over-riding anything we may be able to remove most of these.  Let's test and see what we can remove.

Reply:
Removed unused imports